website/src/main/tut/docs/rules/ImplicitClassPrivateVal.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: ImplicitClassPrivateVal
+---
+
+# ImplicitClassPrivateVal
+
+`val` fields of implicit classes are accessible as an extension. 
```

Comment:
Non-private `val` fields of implicit classes leak as publicly accessible extension methods.